src/Accounts/Accounts/Feedback/OpenAzSurveyLinkCmdlet.cs

```diff
@@ -13,28 +13,58 @@
 // ----------------------------------------------------------------------------------
 
 using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.PowerShell.Common.Share.Survey;
 using Microsoft.WindowsAzure.Commands.Utilities.Common;
 using System;
 using System.Diagnostics;
 using System.Management.Automation;
 using System.Runtime.InteropServices;
+using System.IO;
+using Newtonsoft.Json;
 
 namespace Microsoft.Azure.Commands.Profile.Survey
 {
     [Cmdlet(VerbsCommon.Open, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SurveyLink"), OutputType(typeof(void))]
     public class OpenAzSurveyLinkCmdlet : AzurePSCmdlet
     {
-        private const string _surveyLinkFormat = "https://aka.ms/azpssurvey?Q_CHL=INTERCEPT";
+        private const string _surveyLinkFormat = "https://go.microsoft.com/fwlink/?linkid=2201766&ID={0}&v={1}&d={2}";
+
+        private static string SurveyScheduleInfoFile = Path.Combine(
+            Environment.GetFolderPath(Environment.SpecialFolder.UserProfile),
+            ".Azure", "AzureRmSurvey.json");
 
         protected override IAzureContext DefaultContext => null;
 
         protected override string DataCollectionWarning => null;
 
         public override void ExecuteCmdlet()
-        {
-            WriteInformation(new HostInformationMessage() { Message = $"Opening the default browser to {_surveyLinkFormat}" }, new string[] { "PSHOST" });
-            OpenBrowser(_surveyLinkFormat);
+        {   
+            StreamReader sr = null;
+            DateTime Today = DateTime.UtcNow;
```

Comment:
Use lowercase for local variables
```suggestion
            DateTime today = DateTime.UtcNow;
```